contracts/src/instances/ezeth/EzETHTarget0.sol

```diff
@@ -37,9 +37,15 @@ contract EzETHTarget0 is HyperdriveTarget0, EzETHBase {
     }
 
     /// @notice Gets the ezETH token contract.
-    /// @return The ezETH token contract.
+    /// @return IERC20 The ezETH token contract.
```

Comment:
IERC20 is a type and shouldn’t be in the NatSpec.

Reply:
should this be `address` instead?

Reply:
If you check out other comments in the repo, you'll see what I mean. There isn't a type in any other `@return` statement.